Makefile

```diff
@@ -78,6 +78,11 @@ kind-cni:
 
 # ===========> CLI Targets
 
+cli-dev:
+	@echo "==> Installing consul-k8s CLI tool for ${GOOS}/${GOARCH}"
+	@cd cli; go build -o ./bin/consul-k8s; cp ./bin/consul-k8s ${GOPATH}/bin/
```

Comment:
Nice! I was always doing `go install .` and aliasing `consul-k8s` to `cli`. This is better!